oap-server/server-bootstrap/src/main/resources/trace-sample-rate-setting.yml

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
You add a new file, so you have to change package pom. Otherwise, this file will be in the core jar.

Reply:
https://github.com/apache/skywalking/blob/db2bc935737ee24ad79b6f237b0eaa60991434ef/oap-server/server-bootstrap/pom.xml#L267-L291

Reply:
https://github.com/apache/skywalking/blob/db2bc935737ee24ad79b6f237b0eaa60991434ef/apm-dist/src/main/assembly/binary.xml#L52-L73

https://github.com/apache/skywalking/blob/db2bc935737ee24ad79b6f237b0eaa60991434ef/apm-dist-es7/src/main/assembly/binary-es7.xml#L52-L73